backend/src/routes/leaderboard/mod.rs

```diff
@@ -3,11 +3,16 @@ use actix_web::{HttpResponse, ResponseError};
 use thiserror::Error;
 
 mod chart;
+pub mod sse_handler;
 mod teams;
 
 pub fn config(cfg: &mut utoipa_actix_web::service_config::ServiceConfig) {
     cfg.service(chart::chart);
     cfg.service(teams::teams);
+    cfg.route(
+        "/events",
```

Comment:
I think events may be confusing, because in other places `event` is a ctf that starts at specific time etc. Name it to something like  `/updates`